src/DynamoCore/Models/NodeModel.cs

```diff
@@ -100,7 +100,7 @@ public List<IRenderPackage> RenderPackages
         /// <summary>
         ///     Fired when this NodeModel is disposed.
         /// </summary>
-        public event Action Disposed;
+        public event Action<NodeModel> Disposed;
```

Comment:
This is okay, but I'm assuming you've checked this does not break anything in DynamoRevit.